engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/StandaloneProcessInterpreter.scala

```diff
@@ -77,20 +75,25 @@ object StandaloneProcessInterpreter {
         compileWithCompilationErrors(node, validationContext).andThen(partInvoker(_, nextParts))
       case part@SinkPart(_, endNode, validationContext) =>
         compileWithCompilationErrors(endNode, validationContext).andThen(partInvoker(_, List()))
-      case CustomNodePart(transformer:StandaloneCustomTransformer, node, validationContext, parts, _) =>
-        val result = compileWithCompilationErrors(node, validationContext).andThen(partInvoker(_, parts))
+      case CustomNodePart(transformerObj, node, validationContext, parts, _) =>
+        val validatedTransformer = transformerObj match {
+          case t: StandaloneCustomTransformer => Valid(t)
```

Comment:
why these two cases? backward compatibility?

Reply:
Yep, backward compatibility just like in Flink part. Should I leave here some TODO?